src/main/java/org/myrobotlab/service/config/VertXConfig.java

```diff
@@ -0,0 +1,17 @@
+package org.myrobotlab.service.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VertXConfig extends ServiceConfig {
```

Comment:
there are two of these classes.. one with a capital X and one with a lower case x... please remove/rename one of them as appropriate.

Reply:
deleted the capital X one